src/algorithms/wcc.c

```
@@ -0,0 +1,170 @@
+		info = GrB_Scalar_new(&scalar, GrB_BOOL);
+		ASSERT(info == GrB_SUCCESS);
+
+		info = GxB_Scalar_setElement_BOOL(scalar, true);
+		ASSERT(info == GrB_SUCCESS);
+
+		//info = GxB_Vector_build_Scalar(*N, GrB_ALL, scalar, nrows);
+		//ASSERT(info == GrB_SUCCESS);
+
+		info = GrB_Vector_assign_Scalar(*N, NULL, NULL, scalar, GrB_ALL, nrows, NULL);
+		ASSERT(info == GrB_SUCCESS);
+    
```

Cppcheck (2.10-2)

[error] Common realloc mistake

(memleakOnRealloc)